
try {
    console.log("are ");
    functionadd();
} catch (error) {
    console.log("are you ok");
    console.log(error.message);
}
finally{

    console.log("we will run");  
}